src/extensions/default/DebugCommands/main.js

```diff
@@ -257,6 +273,457 @@ define(function (require, exports, module) {
         brackets.app.showOSFolder(dir);
     }
 
+    function _getDefaultPrefsFilePath() {
+        // Default preferences
+        return defaultPreferencesFullPath;
+    }
```

Comment:
Since `_getDefaultPrefsFilePath` is used only once and is also not exported, it might be better to use `defaultPreferencesFullPath` where `_getDefaultPrefsFilePath` is being used.